app/screens/auth/forgot_password_screen.tsx

```diff
@@ -1,121 +1,135 @@
-import { View, Text, Image, TextInput, TouchableOpacity, StyleSheet, Dimensions } from 'react-native';
-import { useRouter } from 'expo-router';
+import {
+  View,
+  Text,
+  Image,
+  TextInput,
+  TouchableOpacity,
+  StyleSheet,
+  Dimensions,
+} from "react-native";
+import { useRouter } from "expo-router";
 
 // Get screen width and height
-const { width, height } = Dimensions.get('window'); 
+const { width, height } = Dimensions.get("window");
 
 export default function ForgotPasswordScreen() {
-    const router = useRouter();
-    return (
-        <View style = {styles.forgotPasswordScreen}>
-            {/* Background Image */}
-            <Image
-                source = {require('@/assets/images/auth/ForgotPasswordScreen/bg.png')}
-                style = {[styles.backgroundImage]}
-            />
+  const router = useRouter();
+  return (
+    <View style={styles.forgotPasswordScreen}>
+      {/* Background Image */}
+      <Image
+        source={require("@/assets/images/auth/ForgotPasswordScreen/bg.png")}
+        style={[styles.backgroundImage]}
+      />
 
-            {/* Title */}
-            <Text style={styles.titleText}>
-                Forgot your Password ?
-            </Text>
+      {/* Title */}
+      <Text style={styles.titleText}>Forgot your Password ?</Text>
 
-            {/* Column Container */}
-            <View style={styles.colContainer}>
-                {/* Input */}
-                <Text style={styles.text}>Email</Text>
-                <TextInput
-                    style={styles.input}
-                    placeholder="example@your.domain"
-                    placeholderTextColor="#888"
-                    autoComplete='email'
-                    inputMode='email'
-                    keyboardType='email-address'
-                    autoCapitalize='none'
-                    testID='emailInput'
-                />
+      {/* Column Container */}
+      <View style={styles.colContainer}>
+        {/* Input */}
+        <Text style={styles.text}>Email</Text>
+        <TextInput
+          style={styles.input}
+          placeholder="example@your.domain"
+          placeholderTextColor="#888"
+          autoComplete="email"
+          inputMode="email"
+          keyboardType="email-address"
```

Comment:
It's cleaner than the previous version, it goes for what  is written down below too